libraries/cmd/unit_generic.h

```diff
@@ -767,9 +739,19 @@ class Unit : public Armed, public Audible, public Drawable, public Damageable, p
  **** TARGETTING STUFF                                                              ***
  **************************************************************************************
  */
-
 public:
+    Vector GetNavPoint();
+    Unit *Target();
+    const Unit *Target() const;
+    void SetTarget(Unit *target);
     Unit *Threat();
+    const Unit *Threat() const;
+    void SetThreat(Unit *target);
```

Comment:
It's tempting to name this `Threat(Unit *target)` to be consistent with the rest.
However, `unit_generic.cpp:1577` has another function with the same name that does a whole lot more.
Ask me how I know.